<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Visualizza Turno</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xz.qrc">
    <normaloff>:/images/images/AnconAmbiente_Logo.png</normaloff>:/images/images/AnconAmbiente_Logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="annulla_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>500</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
    </property>
    <property name="text">
     <string>INDIETRO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifica_button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>500</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
    </property>
    <property name="text">
     <string>MODIFICA</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="inserimentoturno_tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="dati_tab_2">
     <attribute name="title">
      <string>Dati</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Data e Ora Inizio Turno:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Data e Ora Fine Turno:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Servizio</string>
      </property>
     </widget>
     <widget class="QLabel" name="ora_fine_label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>341</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="ora_inizio_label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Cliente:</string>
      </property>
     </widget>
     <widget class="QLabel" name="tipo_label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="data_fine_label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>400</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="id_servizio_label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>80</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Data Inizio:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>400</y>
        <width>80</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Data Fine:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Tipo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="periodicita_label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>320</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cliente_label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>230</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="data_inizio_label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>360</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="luogo_label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>270</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Luogo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>80</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Periodicit√†:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mezzi_widget_2">
     <attribute name="title">
      <string>Mezzi</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>65</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>MEZZI</string>
      </property>
     </widget>
     <widget class="QLabel" name="mezzo1_label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>15</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabella_mezzi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>681</width>
        <height>401</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>DADADAD</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Modello</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Allestimento</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stato</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="operatori_tab_2">
     <attribute name="title">
      <string>Operatori</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>OPERATORI</string>
      </property>
     </widget>
     <widget class="QLabel" name="operatore1_label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>15</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabella_operatori">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>681</width>
        <height>401</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>170</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cognome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stato</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="xz.qrc"/>
 </resources>
 <connections/>
</ui>
